### DISCUSSION TITLE
Price slider showing twice after applying price filter

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#158] [1. Help]
Hi,

When I apply price filter, It shows bar twice. Any fix for this one?

![image](https://github.com/breezefront/community/assets/132253509/199a92bb-cd72-464d-aec0-d52fbc1a8c90)
![image](https://github.com/breezefront/community/assets/132253509/4192eca0-ab25-4f9c-acf6-9886fc0e07ef)

**Comment by vovayatsyuk** [comment]
Can't reproduce.

[Screencast from 16.06.23 09:50:12.webm](https://github.com/breezefront/community/assets/306080/70160267-efb0-43f9-b7f5-1504b2dab360)
